docker/driver-loader-legacy/docker-entrypoint.sh

```diff
@@ -18,6 +18,28 @@
 #
 
 
+print_usage() {
+	echo ""
+	echo "Usage:"
+	echo "  falco-driver-loader [driver] [options]"
```

Comment:
```suggestion
	echo "  falco-driver-loader [driver] [options]"
```

This is misleading since there's no `falco-driver-loader` executable anymore. 

Still trying to figure out how to fix it. May we print the [container image usage help message](https://github.com/falcosecurity/falco/blob/master/docker/driver-loader-legacy/Dockerfile#L6) instead? :thinking:

Reply:
You are right! Will fix in a follow up PR